neutrinordp/xrdp-neutrinordp.c

```diff
@@ -610,6 +708,31 @@ lxrdp_set_param(struct mod *mod, const char *name, const char *value)
             mod->perf_settings_values_mask |= PERF_DISABLE_CURSOR_SHADOW;
         }
     }
+    else if (g_strcmp(name, "neutrinordp.allow_client_keyboardLayout") == 0)
+    {
+        /* Keyboard values are stored for later processing */
+        mod->allow_client_kbd_settings = g_text2bool(value);
+    }
+    else if (g_strcmp(name, "neutrinordp.overrode_keyboardLayout_mask") == 0)
```

Comment:
Again, I suspect I wasn't being clear in a previous comment.

For the values in xrdp.ini, use `override` instead of `overrode`.

`Override` uses the English present tense, whereas 'overrode' is the past tense.

It's more usual to use the present tense for these settings.